Source/Spark.swift

```diff
@@ -24,31 +24,126 @@ import UIKit
 
 
 /// The main entry point into the SDK. Contains methods for initializing the SDK and verifying developer and end user credentials.
-open class Spark {
+public class Spark {
```

Comment:
You change the Spark from singleton to normal object, does that mean the user can create and use multipal Spark instances in the same process?